vcluster/configure/vcluster-yaml/sync/from-host/configmaps.mdx

```diff
@@ -0,0 +1,152 @@
+---
+title: ConfigMaps
+sidebar_label: configmaps
+sidebar_position: 10
+description: Configuration for ...
+---
+
+import EnableSwitch from '../../../../_partials/config/sync/fromHost/configMaps.mdx'
+import ProAdmonition from '../../../../_partials/admonitions/pro-admonition.mdx'
+import FromHostConfigMapExample from '../../../../_fragments/sync-from-host-configmap-example.mdx'
+
+By default, this is turned off.
+
+:::info No need to configure RBAC
+vCluster automatically adds the required cluster RBAC permissions for retrieving the config maps and syncing the resources from the host to the virtual cluster.
+:::
+
+Enabling this allows you to sync ConfigMaps from the specified namespaces in the host to the specified namespaces in the vCluster.
+
+It is also possible to modify the name of the synced resource in the virtual cluster.
+
+There is no option to sync from all namespaces in the host.
+
+Sync is one-directional, from host to virtual. If you modify an object in the host, vCluster syncs the change to virtual object. When you delete a virtual object, vCluster re-creates it if the host object still exist.
+
+When you delete a host object, vCluster deletes the corresponding virtual object.
+
+Namespaces in the virtual cluster are created automatically during the sync (if they do not exist already).
+
+It is not possible to sync ConfigMaps that were already synced from virtual to host, they are skipped by vCluster.
+
```

Comment:
The above 6 things are pretty random, and don't follow a specific order that makes sense when reading through them. 
Can you write them into a paragraph that follows a sync from beginning to end ?  Starting with the "one-directional" line, and ending with "delete a host object". 
That or put them in a bulleted list of notes.

Reply:
added as a bullet list